.landscape.yaml

```diff
@@ -7,6 +7,7 @@ ignore-paths:
 test-warnings: true
 python-targets:
     - 3
+max-line-length: 200
```

Comment:
yes, either define something that is a strict rule for our project or define nothing at all and comment on pull requests if lines get too long.